packages/number-input/tests/__snapshots__/number-input.test.tsx.snap

```diff
@@ -27,6 +27,7 @@ exports[`should render correctly 1`] = `
   padding-right: 1rem;
   height: 2.5rem;
   border-radius: 0.375rem;
+  vertical-align: top;
```

Comment:
When using `NumberInput` with `size="xs"`, the stepper is slightly shifted down. This fixes it.